scripts/base/frameworks/software/main.zeek

```diff
@@ -238,6 +238,18 @@ function parse(unparsed_version: string): Description
 	return [$version=v, $unparsed_version=unparsed_version, $name=alternate_names[software_name]];
 	}
 
+global parse_cache: table[string] of Description &read_expire=65secs;
```

Comment:
Why `65`? :-)

Reply:
My thought was 'A user-agent is worth keeping in the cache if we see it at least once per minute"

the extra 5 seconds is to avoid an entry expiring just before it is seen again if requests are made exactly every 60 seconds.